### DISCUSSION TITLE
The 'title' property of request parameters is invalid

### DISCUSSION THREAD

**Discussion by Jedore** [fastapi/fastapi#6694] [Questions] [answered]
Python: 3.7.4
FastAPI: 0.65.1

```python
from fastapi import Query, APIRouter

router1 = APIRouter()
@router1.get('/grades/', summary='学段列表')
def query(name_like: str=Query(..., title='名称模糊查询', description='模糊查询')):
    return 
```
But the `title` property is invalid in swagger doc:
![1](https://github.com/Jedore/images/raw/main/fastapi1.png)
![2](https://github.com/Jedore/images/raw/main/fastapi2.png)

So what's the meaning or goal of `title` ？

**Comment by Acerinth** [comment, marked as answer]
Hey there 👋🏼 

Seems like "Title" is currently not in use in Swagger (https://swagger.io/specification/), but it is used in ReDoc. So, if you check out the automated docs in Redoc (`/redoc`), *Title* should be there in parenthesis next to the query parameter name.

Just my assumption.

**Comment by Jedore** [comment]
> Hey there 👋🏼
> 
> Seems like "Title" is currently not in use in Swagger (https://swagger.io/specification/), but it is used in ReDoc. So, if you check out the automated docs in Redoc (`/redoc`), _Title_ should be there in parenthesis next to the query parameter name.
> 
> Just my assumption.

Yeah, you're right. Thanks!
